gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsGlobalStorageStatistics.java

```diff
@@ -228,10 +230,8 @@ protected void addTotalTimeStatistic(String statistic) {
     String parentMeanKey = getMeanKey(parentCounterKey);
 
     assert (means.containsKey(parentMeanKey) && opsCount.containsKey(parentCounterKey));
```

Comment:
we don't need counterKey assert, correct?

Reply:
Already existing code. I guess more check wont hurt.